partner_certificate_of_conduct/tests/test_partner_certificate_of_conduct.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestPartnerCertificateOfConduct(TransactionCase):
+    def setUp(self):
+        super(TestPartnerCertificateOfConduct, self).setUp()
+        self.main_partner = self.env.ref('base.main_partner')
+        self.partner_id_category = self.env.ref(
+            'partner_certificate_of_conduct.id_category_certificate_of_conduct'
+        )
+
+    def test_01_id_number_new(self):
+        # Good CoC
+        vals = {
+            'name': '56048785',
+            'category_id': self.partner_id_category.id,
+        }
+        self.main_partner.write({'id_numbers': [(0, 0, vals)]})
+        id_number = self.main_partner.id_numbers[0]
+
+        self.assertEqual(id_number.name, '56048785')
+
+    def test_02_id_number_duplicate(self):
+        # Duplicate CoC
+        vals = {
```

Comment:
duplicate verifys the same thing as the first one. Is the expected behaviour to have only one id_number instead of two the same? 
in that case must verify len(self.main_partner.id_numbers) == 1

on line 35 you just verify  self.main_partner.id_numbers[0] which we already knew was 56048785